guidelines/sc/21/reflow.html

```diff
@@ -9,7 +9,7 @@ <h4>Reflow</h4>
        <li>Vertical scrolling content at a width equivalent to 320 <a>CSS pixels</a>;</li>
        <li>Horizontal scrolling content at a height equivalent to 256 <a>CSS pixels</a>.</li>
    </ul>
-   <p>Except for parts of the content which require two-dimensional layout for usage or meaning.</p>
+   <p>Except for parts of the content with <a>two-dimensional layout</a> where scrolling in two dimensions is <a>essential</a> for usage or meaning.</p>
```

Comment:
I like the clarity, but it would make the criterion more stringent than precedent established in the Understanding. Something is "essential" when "information and functionality cannot be achieved in another way that would conform." Two-dimensional scrolling of most data tables is not "essential" because the information could be achieved through a [responsive table](https://medium.com/allenhwkim/mobile-friendly-table-b0cb066dbc0e) technique. Two-dimensional scrolling of most toolbars could be avoided with responsive menus.

With this PR do you intend to establish a more stringent precedent for WCAG 2.1-2.2, or just steer the ship in a clearer direction for 3.0? If the former — I might support such a clarification, if it gets enough attention and support from the community. It would need to happen in the Understanding doc as a clarification of what "required" means, not as a change to normative WCAG.